tensorboard/plugins/mesh_visualizer/tf_mesh_dashboard/mesh-loader.js

```diff
@@ -139,7 +139,7 @@ Polymer({
    * Function to call when component must be reloaded.
    */
   reload: function() {
-    if (!this.active) {
```

Comment:
Did you consider guarding for `this._dataProvider` itself, rather than
the lifecycle methods? No change necessarily requested; mostly wondering
if there’s a reason for the indirection.

Reply:
I actually did think about that but I am torn.

On one hand: `reload` does make much sense without component being mounted.
On the other hand, both `active` and `_dataProvider` should be part of the observer AND guard against the undefinedness (in Polymer2+, there is no guarantee that all props are initialized before invoking the observer).

Reply:
Decided to take the second "hand" :)